index.js

```diff
@@ -1,29 +1,46 @@
 import js from '@eslint/js';
 
+// will be removed by ESLint in a future version
+// see: https://eslint.org/blog/2023/10/deprecating-formatting-rules/
+const deprecated = {
```

Comment:
Thanks for separating this out like this — makes it really easy to review and then later revisit 👌